service_container/tags.rst

```diff
@@ -564,3 +564,242 @@ application handlers.
                 ->addTag('app.handler', ['priority' => 20]);
 
     Note that any other custom attributes will be ignored by this feature.
+
+
+Tagged Services Collection with Index
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+What if you want to retrieve a specific service within the injected collection previously?
+But you doesn't want to walk through the whole collection. The solution is to use the ``index_by``
```

Comment:
As it is now, the feature allows defining the key of the iterator: it doesn't allow accessing a specific key without walking the collection (`iteraror_to_array` *does* walk through the collection.)
We should wait for allowing indexed service locators as this should have a direct impact on the wording here.

Reply:
I''l remove `But you doesn't want to walk through the whole collection.`